### ISSUE TITLE
home and end keys do not change the selection

### ISSUE THREAD

**Issue by okonomiyaki** [select2/select2#3739]
In a regular select box, pressing home or end changes the selection to the first or last option, respectively, whether the selection list is expanded or not.

In select2 4.0, home and end keys aren't handled and the event is passed up to the browser.

**Comment by kevin-brown** [comment]
Even though the title issue may not reflect it, this is effectively a duplicate of .https://github.com/select2/select2/issues/3472 and is blocked by the issues described in https://github.com/select2/select2/issues/3279#issuecomment-96450069.

**Comment by the-super-coder** [comment]
If this feature is not going to be a part of the official build, is there any way to extend the library and implement this?
